mmv1/products/apigee/terraform.yaml

```
@@ -103,6 +103,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       # the resources needed for the acceptance test.
       name: "apigee_environment_basic_test"
       primary_resource_id: "apigee_environment"
+      primary_resource_name: "getTestProjectFromEnv(), fmt.Sprintf(\"tf-test-apigee-env%s\", context[\"random_suffix\"])"
```

It looks like this is causing issues; it is causing things like this to be built:

```
			{
				ResourceName:      "google_apigee_environment_iam_binding.foo",
				ImportStateId:     fmt.Sprintf("%s/environments/%s roles/viewer", getTestProjectFromEnv(), fmt.Sprintf("tf-test-apigee-env%s", context["random_suffix"])),
				ImportState:       true,
				ImportStateVerify: true,
			},
```

This results in the import id being set to something like `my-project/environments/tf-test-apigee-envasdfklj` but it should be `organizations/tf-testasdfklj/environments/tf-testasdfklj`. I think you could resolve the issue with something like this:

```suggestion
      primary_resource_name: "fmt.Sprintf(\"organizations/tf-test%s\", context[\"random_suffix\"]), fmt.Sprintf(\"tf-test%s\", context[\"random_suffix\"])"
```

---

Let me know if that helps the tests pass for you locally!

---

Thanks, done! I'm still working with my team to setup the permission for me to run tests locally. We can try and see if the tests pass in your system.